### ISSUE TITLE
Spurious shadow under text

### ISSUE THREAD

**Issue by automatedbugreportingfacility** [mozilla/pdf.js#6129]
This is a regression from PR #4521. There's a grey shadow under **45°**.

URL: http://www.leics.gov.uk/sd34.pdf
1.1.215: https://i.imgur.com/s20TmMU.png
0.8.990: https://i.imgur.com/kaOooD2.png

7e7b728b5d7bfd0477a44f70a9afdd9a30cae221 is the first bad commit
commit 7e7b728b5d7bfd0477a44f70a9afdd9a30cae221
Author: Jonas Jenwald
Date:   Wed Mar 26 22:29:04 2014 +0100

```
Assign non-zero width/height to all rectangles (issue 4260)
```

**Comment by Snuffleupagus** [comment]
Actually, there seems to be more to this issue than just a regression.
Comparing the rendering in PDF.js *just* prior to PR #4521 with Adobe Reader, it seems that there is more going on here. As can be seen in the following screen-shot, it appears that in Adobe Reader the area behind **45°** is actually filled with white, whereas in PDF.js it was semitransparent:

![issue6129](https://cloud.githubusercontent.com/assets/2692120/8232535/e067ad60-15d0-11e5-9fad-793a91c51719.png)

**Comment by yurydelendik** [comment]
Screenshot from preview:
![screen shot 2015-06-18 at 9 27 48 am](https://cloud.githubusercontent.com/assets/1523410/8233475/1246f618-159c-11e5-97e8-dfe905a36c12.png)

**Comment by yurydelendik** [comment]
Removing regression tag, since the Adobe Reader and Mac OSX Preview has the similar issue.

**Comment by Snuffleupagus** [comment]
Fixed by PR #12812
